theano/sandbox/gpuarray/tests/test_dnn.py

```diff
@@ -171,7 +171,7 @@ def test_pooling():
         raise SkipTest(dnn.dnn_available.msg)
 
     # 'average_exc_pad' is disabled for versions < 4004
-    if dnn.version() < 4004:
+    if dnn.version(False) < 4004:
```

Comment:
I did all changes. Can you review again and merge if all is good?

On Tue, Apr 12, 2016 at 2:46 PM, Mathieu Germain <notifications@github.com>
wrote:

> In theano/sandbox/gpuarray/tests/test_dnn.py
> <https://github.com/Theano/Theano/pull/4366#discussion_r59431613>:
>
> > @@ -171,7 +171,7 @@ def test_pooling():
> >          raise SkipTest(dnn.dnn_available.msg)
> >
> >      # 'average_exc_pad' is disabled for versions < 4004
> > -    if dnn.version() < 4004:
> > +    if dnn.version(False) < 4004:
>
> As a parameter is not obvious for a function called version, maybe adding
> the keyword argument would be good.
> dnn.version(raises=False)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/4366/files/35673bc1eb425533a879f26c64e89befc39ecd70#r59431613>
>